Ginger/GingerCoreCommon/Repository/NewRepositorySerializer.cs

```diff
@@ -72,7 +72,10 @@ public static object OnNewRepositorySerializerEvent(NewRepositorySerializerEvent
         public  void SaveToFile(RepositoryItemBase ri, string FileName)
         {
             string txt = SerializeToString(ri);
+            Console.WriteLine("Serialize to string");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)